state/movies/reducer.js

```diff
@@ -0,0 +1,44 @@
+import * as types from './types';
+
+const initialState = {
+  trendingList: {
+    loading: false,
+    error: null,
+    list: []
+  }
+};
+
+function trendingMoviesReducer(state = initialState, action) {
```

Comment:
Use arrow function instead of pre-ES6 style

Reply:
Done.